addons/GodotInk/Src/InkStoryImporter.cs

```diff
@@ -88,6 +92,16 @@ private static void InkCompilerErrorHandler(string message, ErrorType errorType)
                 throw new InvalidInkException();
         }
     }
+
+    private class RelativePathFileHandler : Ink.IFileHandler {
```

Comment:
```suggestion
    private class RelativePathFileHandler : IFileHandler {
```

Name can be simplified.

Reply:
Done.

Today I learned that Allman indentation has a name. :)